_client/livethere-app/src/components/summary/rentalSummary/RentalSummary.js

```diff
@@ -177,40 +143,34 @@ const RentalSummary = (props) => {
     </>
   )
 
-  const summaryText = (
-    <div>
+  let summaryText = <div> No listings found! </div>  
+  
+  if (summary && summary.metric === 'average' && summary.rentalsCount > 0){
+    summaryText = (
       <div>
-        {
-          summary && summary.rentalsCount > 0 ? (
-            <div>
-              <div>
-                Average rental price is <b>${summary.average?.toFixed(0)}/mo per room</b>
-              </div>
-              <div>
-                Calculated with <b>{summary.rentalsCount}</b> listings found online
-              </div>
-            </div>
-          ) : <div>No listings found!</div>
-        }
+        <div>
+          Average rental price is <b>${summary.summary?.toFixed(0)}/month per room</b>
+        </div>
+        <div>
+    Calcuated with <b>{summary.rentalsCount}</b> listings found online.
+        </div>
       </div>
+    );
+  }
+  else if(summary && summary.metric === 'prediction'){
+    summaryText = (
       <div>
-        {
-          prediction ? (
-            <div>
-              Predicted rental price is <b>${prediction.prediction?.toFixed(0)}/mo per room</b>
-            </div>
-          ) : null
-        }
+          Predicted rental price is <b>${summary.summary?.toFixed(0)}/month per room</b>
```

Comment:
can we rename the response from backend so it's not summary.summary? maybe summary.estimate or why not just keep it the old way like summary.average?

Reply:
Ill rename it to value. Summary.summary is either average or prediction